packages/contracts-bedrock/src/libraries/L1BlockErrors.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
```

Comment:
You can set this to be more permissive since its just errors like `^0.8.0` in case somebody imports it